jetty-quickstart/src/main/config/modules/jetty-quickstart.d/quickstart-webapp.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "https://www.eclipse.org/jetty/configure_10_0.dtd">
+
+<Configure class="org.eclipse.jetty.webapp.WebAppContext">
+
+  <Call name="setAttribute">
+    <Arg>org.eclipse.jetty.quickstart.ORIGIN_ATTRIBUTE</Arg>
+    <Arg><Property name="jetty.quickstart.origin" default="origin"/></Arg>
```

Comment:
Having these properties set in a ini file to be copied in a $JETTY_HOME/etc XML file to be used here in a $JETTY_BASE/etc/ XML file is a little convoluted.  Now we have QuickStartConfiguration, I'm wondering if we can do better.